aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -795,7 +795,7 @@ SparseTensor sparse_mask(const Tensor& t, const SparseTensor& mask) {
 
   if (t.layout() == at::kSparse) {
     if (!t._nnz()) {
-      auto res = mask.clone().to(t.device(), t.scalar_type());
+      auto res = mask.to(t.device(), t.scalar_type(), false, true);
```

Comment:
Please add '/*param=*/' comments to annotate the positional args for the Boolean values at least.

Reply:
Why add 2 Booleans?

Reply:
First is for non_blocking

Reply:
> Please add '/_param=_/' comments to annotate the positional args for the Boolean values at least.

Good advice.